templates/sonar.properties

```diff
@@ -421,4 +421,4 @@ sonar.log.maxFiles=7
 
 #--------------------------------------------------------------------------------------------------
 # SECRETS
-sonar.secretKeyPath=/opt/sonar/.sonar/sonar-secret.txt
+sonar.secretKeyPath={{ sonar_home }}/.sonar/sonar-secret.txt
```

Comment:
I'm not sure if I'm missing something, but I see the task to install this file in https://github.com/RentTheRunway/sonar/blob/master/tasks/sonar_config.yml#L44-L50 but I don't see the template. Am I going crazy?

Reply:
No, not crazy, I think this got lost during a back-and-forth I had about putting the secret in `ansible` repo or `sonar` repo